appveyor.yml

```diff
@@ -5,6 +5,7 @@ environment:
   DEPS_PREFIX: "C:/projects/nvim-deps/usr"
   # Silence/redirect errors due to missing locking support (via libgcov).
   GCOV_ERROR_FILE: "$(TEMP)/libgcov-errors.log"
+
```

Comment:
might as well undo this empty line to avoid blame noise. This PR should run without "skip ci"/"skip travis" one last time.